### DISCUSSION TITLE
ElTable列如何设置实现类型TinyGrid的插槽

### DISCUSSION THREAD

**Discussion by langren1992** [opentiny/tiny-engine#327] [Q&A]
![图片](https://github.com/opentiny/tiny-vue/assets/10414947/a055541c-1002-4e2e-b59b-3f156edf5cb8)
怎么设置ElTable的属性来实现TinyGrid类似列中允许拖入组件的插槽效果

**Comment by chilingling** [comment]
@langren1992 
Element-plus 组件库 Table 组件不支持配置式 columns，必须手动声明 el-columns，所以这里需要支持表格列默认插槽的话，需要将 columns props 中的 slot 转化为 children。我们研讨一下给个解决方案。
